src/test/kotlin/shared/mothers/IdentifierMother.kt

```diff
@@ -13,4 +13,5 @@ object IdentifierMother {
         return Identifier(uuid)
     }
     const val invalidPrimitiveIdentifier = "InvalidUUID-12345"
+    fun getInvalidIdentifierPrimitive() = invalidPrimitiveIdentifier
```

Comment:
Not needed, you can access it via the constant (invalidPrimitiveIdentifier), a funtion isn't really needed